firmware_motion_board/src/pid_regulator.c

```diff
@@ -0,0 +1,56 @@
+#include "pid_regulator.h"
+
+void PidReg_update(PidReg_t* reg)
+{   
+    // P, I, D, terms discretized with Backward Euler (s = (z - 1) / zT)
+    float err_old = reg->err;

```

Comment:
```suggestion
    const float err_old = reg->err;

```
Ako se promjenjiva ne mijenja mnogo se lakse cita sa `const` prefiksom

Reply:
Reseno.